src/PeerReview/Types.hs

```diff
@@ -53,11 +57,11 @@ data ReviewStatus = Waiting
                   | Accepted
                   deriving (Show)
 
-data ReviewTask = ReviewTask
-    { rtContent :: Text
-    , rtTaskId  :: TaskID
-    , rtUserId  :: UserID
-    } deriving (Show)
+data Task = Task
+    { tId      :: TaskID
```

Comment:
Yeah, I think we can.. If we think of real life, each task has some title, identifier, or description to identify the task itself.

Another option would be to separate these two to `Task` and `TaskSubmission`